test/contracts/ownable/ownable-events.ts

```diff
@@ -1,3 +1,6 @@
+/* tslint:disable */
```

Comment:
Nothing changed in this file and the pre-commit linter run is complaining many things about this file